realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -185,13 +184,12 @@ boolean hasAssetFile() {
     }
 
     /**
-     * Returns input stream object to the Realm asset file.
+     * Returns the path to the Realm asset file.
      *
-     * @return input stream to the asset file.
-     * @throws IOException if copying the file fails.
+     * @return absolute path to the asset file.
```

Comment:
Is this true? Historically it is just the path for files in assets, and I believe that is a zipped file, so talking about "absolute path" doesn't feel correct?